src/Resolver/Value/InputValue.php

```diff
@@ -48,17 +48,23 @@ public function getRawValue() : \stdClass
         return $return;
     }
 
-    public function printValue() : string
+    public function printValue(bool $prettyPrint = false) : string
     {
         $component = [];
 
         foreach ($this->value as $key => $value) {
             \assert($value instanceof ValidatedValue);
 
-            $component[$key] = $key . ':' . $value->printValue();
+            $component[$key] = $key . ($prettyPrint ? ': ' : ':') . $value->printValue($prettyPrint);
         }
 
-        return '{' . \implode(',', $component) . '}';
+        if (!$component) {
+            return '{}';
+        }
+
+        return $prettyPrint
+            ? '{' . \PHP_EOL . "\x20\x20\x20\x20" . \implode(',' . \PHP_EOL . "\x20\x20\x20\x20", $component) . \PHP_EOL . "\x20\x20" . '}'
```

Comment:
Why is there \x20 instead of normal space?

Reply:
Also, what if there is some inner object, wouldn't indention be different?

Reply:
And also create test for introspection, which validates default value when prettyprint is disabled.